### PR TITLE
Removed `pallet::getter` usage from `pallet-collective`

### PR DESCRIPTION
**Pull request by muraca**
Part of #3326 
This one is easier as all the storage items are public. 

@ggwpez @kianenigma @shawntabrizi

### CODE REVIEW

**Comment by muraca** [comment]
In the `prdoc` i didn't mention `pallet-alliance`, where I only had to update tests. Should I?

**Comment by liamaharon** [comment]
> In the prdoc i didn't mention pallet-alliance, where I only had to update tests. Should I?

Not necessary, doesn't impact anyone downstream

File: prdoc/pr_3456.prdoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: removed `pallet::getter` from `pallet-collective`
```
Comment (liamaharon) [inline prdoc/pr_3456.prdoc:4]:
```suggestion
title: Removed `pallet::getter` usage from `pallet-collective`
```

File: prdoc/pr_3456.prdoc [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR removes all the `pallet::getter` usages from `pallet-collective`, and updates depdendant runtimes accordingly.
```
Comment (liamaharon) [inline prdoc/pr_3456.prdoc:9]:
```suggestion
      This PR removes `pallet::getter` usage from `pallet-collective`, and updates dependant code accordingly.
```

**Review by liamaharon — APPROVED** [review]

**Comment by liamaharon** [comment]
bot fmt

**Review by kianenigma — APPROVED** [review]

**Comment by muraca** [comment]
Thanks! I will be working again on #1445 in the next few days

**Pull request merged**